.ci/code_generation.py

```diff
@@ -8,6 +8,9 @@
 from ansys.dpf import core
 from ansys.dpf.core.operators import build
 
+core.set_default_server_context(core.AvailableServerContexts.premium)
+core.start_local_server(config=core.AvailableServerConfigs.LegacyGrpcServer)
```

Comment:
This prevents deletion of operators if for whatever reason we failed to start a DPF server.